src/porepy/applications/test_utils/partial_discretization.py

```diff
@@ -0,0 +1,37 @@
+"""Utility functions for partial discretization updates."""
+
+import numpy as np
+
+import porepy as pp
+
+
+def perform_partial_discretization_specified_nodes(
```

Comment:
I am unsure what partial discretization means. We should consider better documentation or rename the file.

Reply:
I will improve the documentation to make things a bit clearer.